onnx/test/shape_inference_test.py

```diff
@@ -173,6 +194,34 @@ def _compare_value_infos(
                 "Unrecognized value info type in _compare_value_infos: ", str(vi_type)
             )
 
+    def _test_an_op(self, op_name, test_function_name) -> None:
+        """Given an operator name and test_function_name, run tests for op versions > 5.
+        :param op_name: The operator name.
```

Comment:
Prefer google style comments.

Reply:
Done. Moved the comment to `single_op_all_version_tester`.